lib/node_modules/@stdlib/stats/base/dnanvarianceyc/src/main.c

```diff
@@ -34,45 +35,57 @@
 * @param correction  degrees of freedom adjustment
 * @param X           input array
 * @param stride      stride length
-* @return            output value
+* @return            variance
```

Comment:
```suggestion
* @return            output value
```
no need for changes here IMO